### DISCUSSION TITLE
Kedro compatibility: add a `to_string` method

### DISCUSSION THREAD

**Discussion by luizvbo** [datapane/datapane#199] [Ideas]
I am using datapane to store my reports in my Kedro pipelines. However, I would like to use datapane such that Kedro would be responsible for storing the report (and not datapane). Thus, I would need to pass a string containing the full HTML file content to Kedro.

I think this could be done by separating the logic for rendering the file from the method `write` in `datapane/src/datapane/client/api/report/core.py`.

**Comment by shahinrostami** [comment]


**Reply by luizvbo** [reply]
Ok. Thank you.

**Comment by shahinrostami** [comment]


**Reply by luizvbo** [reply]
Thanks @shahinrostami! I have been using the method for a few weeks now.
